sw/device/silicon_creator/lib/drivers/flash_ctrl.c

```diff
@@ -536,14 +520,11 @@ void flash_ctrl_info_cfg_set(flash_ctrl_info_page_t info_page,
       reg, FLASH_CTRL_BANK0_INFO0_PAGE_CFG_0_EN_0_FIELD, kMultiBitBool4True);
   reg = bitfield_field32_write(
       reg, FLASH_CTRL_BANK0_INFO0_PAGE_CFG_0_SCRAMBLE_EN_0_FIELD,
-      cfg.scrambling == kMultiBitBool8True ? kMultiBitBool4True
-                                           : kMultiBitBool4False);
+      cfg.scrambling);
```

Comment:
@tjaychen just to confirm that this behavior didn't change: Any value other than `0x6` (`kMultiBitBool4True`) will disable these and won't cause any issues, right?

Asking because this function is called from `flash_ctrl_init()` with values read from OTP, which will not be `0x9` or `0x6` until OTP is programmed. We would like to keep these disabled unless explicitly enabled in OTP.

Reply:
This behavior did not change. We are using the `mubi4_test_true_strict(...)` comparison function under the hood for these MUBI CSRs, which is essentially a test with equality against `kMultiBitBool4True`. Hence, any other value will disable the corresponding feature.